### PR TITLE
Add rvc3

### PR DESCRIPTION
**Pull request by HonzaCuhel**
image

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -108,6 +110,13 @@ function App() {
                           Have trouble picking the right version? See <a href="https://docs.google.com/spreadsheets/d/16k3P-LxPMFREoePLvoLqDZo0Xu_tRcSpm_BjQE3PHQY/edit?usp=sharing" target="_blank">here</a> for the version overview.
                         </p>
                       </div>
+                      <div className="mb-3 btn-group" role="group" aria-label="Basic radio toggle button group">
```
Comment (tersekmatija) [inline client/src/App.js:113]: Let's make them equal width.

**Review by tersekmatija — CHANGES_REQUESTED** [review]
In blobconverter, version for RVC3 should also be set to `"2022.3_RVC3"` instead of `"2022.1"` in all files if `not self.use_rvc2`

**Comment by HonzaCuhel** [comment]
image

Analysis of RVC2 conversion using OpenVino 2022.3.0:
- Yolov6tr1 (416x416px) is a bit faster, I've measured `24.72 ± 0.09FPS` vs. previously `23.76 ± 0.67 FPS` 
- Yolov6nr3 (416x416px) is a bit slowe, I've measured `47.74 ± 0.26FPS` vs. previously `50.31 ± 0.17 FPS` 
- Yolov7tiny (416x416px) is a bit faster, I've measured `43.16 ± 1.72FPS` vs. previously `41.22 ± 0.16 FPS` 
- Yolov8n (416x416px) is a bit slower, I've measured `29.02 ± 0.15FPS` vs. previously `31.20 ± 0.11FPS`

File: main.py [hunk 2]
```diff
@@ -127,7 +131,8 @@ async def upload_file(request):
         exporter.export_blob()
     except Exception as e:
         sentry_sdk.capture_exception(e)
-        raise ServerError(message="Error while converting to blob", status_code=523)
+        raise ServerError(message="Error while exporting to blob (this may be caused by trying to using the RVC3 export in which we are experiencing issues, we are working on them).", status_code=526)
```
Comment (tersekmatija) [inline main.py:134]: Maybe better reword this.

File: yolo/exporter.py [hunk 3]
```diff
@@ -89,9 +92,11 @@ def export_blob(self):
             bin=str(self.f_bin.resolve()),#as_posix(),
             data_type="FP16",
             shaves=self.n_shaves,
-            version="2022.1",
+            version="2022.1" if self.use_rvc2 else "2022.3_RVC3",
             use_cache=False,
-            output_dir=self.conv_path.resolve()
+            output_dir=self.conv_path.resolve(),
+            url="https://blobconverter.luxonis.com" if self.use_rvc2 else \
```
Comment (tersekmatija) [inline yolo/exporter.py:98]: Would probably be good if this is read from ENV variable. We'll likely migrate dev-blobconverter to the main blobconverter soon. And ideally we don't need it hardcoded so it's an easy change.

File: yolov7/yolo/exporter.py [hunk 4]
```diff
@@ -89,9 +91,11 @@ def export_blob(self):
             bin=str(self.f_bin.resolve()),#as_posix(),
             data_type="FP16",
             shaves=self.n_shaves,
-            version="2022.1",
+            version="2022.1" if self.use_rvc2 else "2022.3_RVC3",
             use_cache=False,
-            output_dir=self.conv_path.resolve()
+            output_dir=self.conv_path.resolve(),
+            url="https://blobconverter.luxonis.com" if self.use_rvc2 else \
```
Comment (tersekmatija) [inline yolov7/yolo/exporter.py:97]: Same here.

**Review by tersekmatija — APPROVED** [review]
LGTM, some small changes requested. Then feel free to deploy. Thanks!

File: main.py (hunk 2 above)
Comment (HonzaCuhel) [reply main.py:134]: Do you have any suggestions how to reword it, please?

Comment (tersekmatija) [reply main.py:134]: "Error when exporting to blob, likely due to certain operations being unsupported on RVC3. If interested in further information, please open a GitHub issue."

Something along those lines?

Comment (HonzaCuhel) [reply main.py:134]: Sounds good to me!

**Pull request merged**